src/conf.c

```diff
@@ -0,0 +1,280 @@
+//
+// Created by Pierre Delmer on 23/10/2018.
+//
+
+#include "conf.h"
+
+/**
+ * Load the configuration array
+ * @param char *filename
+ * @param arraylist_t *conf
+ */
+void load_configuration(char *filename, arraylist_t *conf) {
+    if (strlen(filename) <= 0) return;
+
+    FILE *src;
+    char *line;
+    char *tmp;
+    rule_t *e;
+
+    src = fopen(filename, "rb");
+    if(src == NULL) return;
+
+    line = malloc(sizeof(char) * 255);
+    tmp = malloc(sizeof(char) * 255);
+    e = malloc(sizeof(rule_t));
+
+    exclude_conf_file(conf, filename);
+
+    while(fgets(line, 255, src) != NULL){
+        if(strstr(line, "=extends") != NULL){
```

Comment:
Should compare `strstr` result to `line` so you get if the line **starts with** `=extends` and not **contains** `=extends`